packages/onboarding/src/sensei-step-container/styles.scss

```diff
@@ -0,0 +1,27 @@
+.sensei {
+	.signup-header {
+		display: none;
+	}
+
+	.step-container {
+		max-width: 100%;
+	}
+
+	.progress-bar {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 10px;
+		z-index: 10;
+		border-radius: 0;
+
+		&__progress {
+			border-radius: 0;
+		}
+	}
+
+	* {
+		box-sizing: border-box;
+	}
+}
```

Comment:
Overall I'd be careful about dramatic spacing modifications to `StepContainer` as these seem fragile; someone changing the `StepContainer` in future for sure won't test Sensei flow.

Worth cloning to different HTML structure then even?

Not a blocker!

Reply:
I moved the selector inside our own components in d31a0e0fd3631924d3f29358fca4a65d4e937d5e to mitigate the risk.